### ISSUE TITLE
lif2: assign owner to community multi-sig

### ISSUE THREAD

**Issue by mfw78** [windingtree/lif2#3]
**tl;dr**
Superuser functions such as `pause()`, `stop()` are assigned to an EOA (nominally the Winding Tree Deployer). This represents immediate medium level security risks.

**Overview**

The Lif2 implementation contains two types of superuser:

1. `ProxyAdmin` - this is responsible for the token logic of the contract, and represents essentially root level access to the contract state by virtue of token logic modification. This is guarded by the WT Community multi-sig `0x876969b13dcf884C13D4b4f003B69229E6b7966A`.

2. `owner` - this is an account stored in state for the contract that allows `pause()` and `stop()` on the contract - used for pausing the contract in it's entirety or stopping the claimable period. **This is currently set the Winding Tree Deployer `0x008235a1Ed13130861b82DCF21FB2E39396F0673`.**

Hypothetically should the deployer EOA key be compromised, there would be no loss of funds, but there would be a denial of service, attack vector, freezing all token holder's balance of LIF.

Therefore it is **strongly recommended** to immediately execute the following:

`Lif2.transferOwnership(0x876969b13dcf884C13D4b4f003B69229E6b7966A)`

**Comment by mfw78** [comment]
```js
{
  chainId: 1,
  data: '0xf2fde38b000000000000000000000000876969b13dcf884c13d4b4f003b69229e6b7966a',
  from: '0x008235a1ed13130861b82dcf21fb2e39396f0673',
  to: '0x9c38688e5acb9ed6049c8502650db5ac8ef96465',
  value: { BigNumber: "0" },
}
```

**Comment by mfw78** [comment]
Ownership transferred to community multi-sig. Refer: https://etherscan.io/tx/0x34fcc4d6871804adcde0e43bfae8f06e5404f69f174879af405893845e809cc3
